src/Config/Schema/Formatter.php

```diff
@@ -39,14 +39,18 @@ public function format(array $data): string
         $text = '';
 
         foreach ($data as $key => $item) {
+            if (isset($item[Schema::SCHEMA_DUMP]) && !$item[Schema::SCHEMA_DUMP]) {
```

Comment:
`isset()` should be eliminated in favor to `!empty()` only

Reply:
`!empty` doesn't work in this case. We need to check if option configured and it `false`

Reply:
done